src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessThreadTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Security;
+using System.IO;
+using Xunit;
+using System.Threading;
+using System.Linq;
+
+namespace System.Diagnostics.ProcessTests
+{
+    public class ProcessThreadTests : ProcessTestBase
+    {
+        [Fact]
+        public void TestCommonPriorityAndTimeProperties()
+        {
+            ProcessThreadCollection threadCollection = _process.Threads;
+            Assert.True(threadCollection.Count > 0);
+
+            ProcessThread thread = threadCollection[0];
+
+            if (ThreadState.Terminated != thread.ThreadState)
+            {
+                Assert.Equal(_process.BasePriority, thread.BasePriority);
+                Assert.True(thread.CurrentPriority >= 0, "Thread CurrentPriority");
+                Assert.True(thread.PrivilegedProcessorTime.TotalSeconds >= 0, "Thread PrivilegedProcessorTime");
+                Assert.True(thread.UserProcessorTime.TotalSeconds >= 0, "Thread UserProcessorTime");
+                Assert.True(thread.TotalProcessorTime.TotalSeconds >= 0, "Thread TotalProcessorTime");
+            }
+        }
+
+        [Fact]
+        public void TestThreadCount()
+        {
+            Process xunitCurrentProcess = Process.GetCurrentProcess();
+
+            // The current process will be the xunit process, which creates threads for each test collection. 
+            // There should be atleast one thread active to represent the current test class, at the time of execution of this test.
+            Assert.True(xunitCurrentProcess.Threads.Count >= 1);
```

Comment:
At least two should be safe in any .NET process I believe.  That said, this is why I suggesting previously that you actually spin up a known number of threads and then verify the count is at least that, e.g.
```C#
int numThreads = 20;
var ce = new CountdownEvent(numThreads);
var mre = new ManualResetEventSlim();
for (int i = 0; i < numThreads; i++)
{
    new Thread(() => {
        ce.Signal();
        mre.Wait();
    }) { IsBackground = true }.Start();
}
ce.Wait();
Assert.True(Process.GetCurrentProcess().Threads.Count > numThreads);
mre.Set();
```
or something like that.